src/dynamics/video_player/player/player.js

```diff
@@ -645,6 +649,8 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
 
                             if (floatingWidth) styles.width = isNaN(floatingWidth) ? floatingWidth : parseFloat(floatingWidth).toFixed(2) + "px";
                             if (floatingHeight) styles.height = isNaN(floatingHeight) ? floatingHeight : parseFloat(floatingHeight).toFixed(2) + "px";
+
+                            containerStyles = styles;
```

Comment:
Hi @sambua, the container isn't supposed to be changed during floating, we shouldn't set floatingHeight or floatingWidth to it. This causes issues when player starts in floating position.
```suggestion
```

Reply:
Hi @ntshcalleia yes, I knew about it, I've reviewed before this chnage. 

But I've put this line back and tested and seems all works as expected. Before this line was out of condition. 

Also without it, when you're on floating and go fullscreen, after you go back from fullscreen floating position changes it's initial position.